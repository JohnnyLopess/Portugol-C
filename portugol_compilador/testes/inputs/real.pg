programa
{
    funcao inicio() {
        real numero
        numero = 3.14
        escreva(numero)
    }
}
