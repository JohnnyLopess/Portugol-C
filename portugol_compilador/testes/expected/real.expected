#include <stdio.h>

int main() {
    float numero;
    numero = 3.14;
    printf("%f\n", numero);
    return 0;
}
